zlib.js

```diff
@@ -1,452 +0,0 @@
-// Copyright Joyent, Inc. and other Node contributors.
```

Comment:
same as the fs file this is an old version of zlib rewritten to include readable streams, looks like it was maybe part of a half finished effort to include all the places in the stdlib that included streams